asyncsleepiq/fuzion/sleeper.py

```diff
@@ -22,9 +22,11 @@ async def set_sleepnumber(self, setting: int) -> None:
 
     async def fetch_sleepnumber(self) -> None:
         """Update fav_sleep_number from API."""
-        result = await self.api.bamkey(self.bed_id, "GetSleepNumberControls")
-        updating, current, last = result.split()
-        self.sleep_number = int(current)
+        args = [SIDES_FULL[self.side].lower()]
+        result = await self.api.bamkey(self.bed_id, "GetSleepNumberControls", args=args)
+        isUpdating, ambientNumber, userNumber = result.split()
+        print("CURRENT NUMBER FOR", self.side, result)

```

Comment:
Remove prints

Reply:
My b - I removed in 48e50bee56e19fd27b2d0b6359abafa3cc63b599!